content/en/docs/tasks/security/authz-tcp/index.md

```diff
@@ -71,32 +82,50 @@ to the `v2` version, which talks to a MongoDB backend using TCP.
 
     On the product page, you can see an error message on the **Book Reviews** section.
     The message reads: **"Ratings service is currently unavailable."**. The message appears because we
-    switched to use the `v2` subset of the `ratings` service without deploying the MongoDB service.
+    switched to use the `v2` subset of the `ratings` workload without deploying the MongoDB workload.
+
+1. Deploy the MongoDB workload:
+
+    {{< tabset cookie-name="sidecar" >}}
+
+    {{< tab name="Automatic sidecar injection enabled" cookie-value="auto" >}}
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo-db.yaml@
+    {{< /text >}}
 
-1. Deploy the MongoDB service:
+    {{< /tab >}}
 
-    * To deploy MongoDB in a cluster **with** automatic sidecar injection enabled:
+    {{< tab name="Manual sidecar injection" cookie-value="manual" >}}
 
-        {{< text bash >}}
-        $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo-db.yaml@
-        {{< /text >}}
+    {{< text bash >}}
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/bookinfo-db.yaml@)
+    {{< /text >}}
 
-    * To deploy MongoDB in a cluster **without** automatic sidecar injection enabled:
+    {{< /tab >}}
 
-        {{< text bash >}}
-        $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/bookinfo-db.yaml@)
-        {{< /text >}}
+    {{< /tabset >}}
 
 1. Go to the Bookinfo product page at `http://$GATEWAY_URL/productpage`.
 
 1. Verify that the **Book Reviews** section shows the reviews.
 
-## Enabling Istio authorization
+## Applying a default `deny-all` policy on the workload
```

Comment:
Why is this a heading? The content still falls under "Installing and configuring a TCP workload"
Do not use headings to emphasize text or steps. Headings provide content structure and categorization, not emphasis. You can always add explanatory text to a list of steps, you just need to indent it accordingly. Applying the `deny-all` policy is not an independent procedure needing its own section. It is a step towards your stated goal and should be handled accordingly in the text to avoid confusing readers.

```suggestion
   With the MongoDB workload deployed and before we configure authentication to only allow authorized requests,
   we need to apply a default `deny-all` policy for the workload to ensure that all requests to the MongoDB
   workload are denied by default. 
```